src/main/java/biblioteket/roborally/programcards/CardComparator.java

```diff
@@ -0,0 +1,12 @@
+package biblioteket.roborally.programcards;
+
+import java.util.Comparator;
+
+public class CardComparator implements Comparator<ICard> {
```

Comment:
Er det en grunn til at disse er `Comparator` istedenfor å gjøre at `Card` implementerer `Comparable`? (spør av egen interesse)

Reply:
Googlet litt rundt for å få den til å funke med collections.sort og arrays.sort, virket som om dette var standard. Kan også fikse sånt som at null blir sortert til venstre.

Reply:
Prøvde nå, Comparable funker ikke for å sortere nullverdier, som gjøres med program registeren